warehouse/templates/search/results.html

```diff
@@ -178,9 +178,9 @@ <h2 class="sr-only">{% trans %}Search results{% endtrans %}</h2>
             <input id="search" type="hidden" name="q" value="{{ term }}">
             <label for="order">{% trans %}Order by{% endtrans %} &nbsp;</label>
             <select class="-js-form-submit-trigger" id="order" name="o">
-              {{ search_option(gettext("Relevance"), "") }}
-              {{ search_option(gettext("Date last updated"), "-created") }}
-              {{ search_option(gettext("Trending"), "-zscore") }}
+              {{ search_option("Relevance", "") }}
```

Comment:
will removing the `gettext` "untranslated" these strings?

Reply:
Woops, yeah, that wasn't intentional, good catch 👍